client/src/graphql/mutations.js

```diff
@@ -97,3 +97,23 @@ export const CREATE_PUBLICATION = gql`
     }
   }
 `;
+
+export const CREATE_SUBMISSION = gql`
+  mutation CreateSubmission(
```

Comment:
We will make a new ZenHub ticket to address the simultaneous mutation for the submissions pivot table.